app/assets/Components/Map/MapStyle.js

```diff
@@ -0,0 +1,511 @@
+export const mapStyle = [
+  // {
```

Comment:
I was planning to leave it in there for now in case I changed my mind about not wanting it, do you think I should delete it when I'm not completely sure I'm not going to want it?